app/src/cambodia/java/utils/PhoneMask.java

```diff
@@ -0,0 +1,55 @@
+package utils;
+
+/**
+ * Created by ina on 02/08/2016.
+ */
+public class PhoneMask {
+
+
+
+    /**
+     * Formatted telephone mask: 0NN NNN NNN{N}
+     */
+    public static final String FORMATTED_PHONENUMBER_MASK = "0\\d{2} \\d{3} \\d{3,4}";
+
+    /**
+     * Formatted telephone mask: 0NN NNN NNN{N}
```

Comment:
@idelcano Can you change comments in method headers? I got the impression some of them where just copy & paste, maybe I am wrong though